File: app/src/main/java/io/github/erikjhordanrey/kata_dagger2/view/adapter/CharacterAdapter.java
package io.github.erikjhordanrey.kata_dagger2.view.adapter;

import androidx.cardview.widget.CardView;

public interface CharacterAdapter {

    int MAX_ELEVATION = 8;

    float getElevation();

    CardView getCardViewAt(int position);

    int getCount();
}

File: app/src/main/java/io/github/erikjhordanrey/kata_dagger2/view/adapter/CharacterAdapter.kt
package io.github.erikjhordanrey.kata_dagger2.view.adapter

import androidx.cardview.widget.CardView

interface CharacterAdapter {
    val elevation: Float
    fun getCardViewAt(position: Int): CardView?
    fun getCount(): Int

    companion object {
        const val MAX_ELEVATION = 8
    }
}
